Resources/Prototypes/_Pirate/Entities/Structures/Doors/Airlocks/airlocks.yml

```
@@ -6,3 +6,21 @@
+  - type: Sprite
+    sprite: _Pirate/Structures/Doors/Airlocks/Standard/interdyne.rsi
+    layoutId: AirlockInterdyne
+    
+- type: entity
+  parent: AirlockGlass
+  id: AirlockInterdyneGlass
+  suffix: Interdyne
+  components:
+  - type: Sprite
+    sprite: _Pirate/Structures/Doors/Airlocks/Glass/interdyne.rsi
+    layoutId: AirlockInterdyne
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)